docs/superpowers/plans/2026-03-25-foundation-design-system.md

```
@@ -0,0 +1,1377 @@
+
+- Create: `convex/schema.ts`
+- Create: `convex/tsconfig.json`
+- Create: `convex/auth.config.ts`
+- Create: `.env.local`
+- Modify: `.gitignore`
+
+- [ ] **Step 1: Add `convex/_generated` to .gitignore**
+
+Append to `.gitignore`:
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)